BitmarkRegistry/Extensions/UIViewController+Extension.swift

```diff
@@ -70,4 +70,12 @@ extension UIViewController {
       handler(photoAuthorizationStatus)
     }
   }
+
+  func doWhenConnectedNetwork(completion: () -> Void) {
+    if NetworkManager.isReachable() {
+      completion()
+    } else {
+      showAlert(title: "Network Error", message: Constant.Error.networkError)
```

Comment:
I'd suggest to use a banner instead of showing an alert.
See the [UX](https://github.com/SwiftKickMobile/SwiftMessages)

Reply:
Or [this](https://github.com/Daltron/NotificationBanner)